src/main/java/stringCalculator/OperatorWith.java

```diff
@@ -0,0 +1,21 @@
+package stringCalculator;
+
+public class OperatorWith {
+    double a;
+    double b;
+    Operator operator;
+
+    protected OperatorWith(double a, Operator operator, double b) {
+        this.a = a;
+        this.operator = operator;
+        this.b = b;
+    }
+
+    public double calculate() {
+        return operator.calculate(a, b);
+    }
+
+    public static OperatorWith of(double a, Operator operator, double b) {
+        return new OperatorWith(a, operator, b);
+    }
```

Comment:
```suggestion
    public static OperatorWith of(double a, Operator operator, double b) {
        return new OperatorWith(a, operator, b);
    }
    
    public double calculate() {
        return operator.calculate(a, b);
    }
```

클래스 구현 순서는 아래를 참고해주세요~

```
class A {
    상수(static final) 또는 클래스 변수
    인스턴스 변수
    생성자
    팩토리 메소드
    메소드
    기본 메소드 (equals, hashCode, toString)
}
```